### ISSUE TITLE
[TW-149] task purge

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#202]
_Dirk Deimeke on 2011-06-28T14:49:44Z says:_ 

 Please introduce a "task purge" command, which cleans the data files from any deleted items.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2011-06-28T14:49:44Z
Modified: 2015-03-14T16:42:35Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2015-03-14T16:42:35Z says:_ 

 Closing this. In the interest of keeping the Taskwarrior core lean, there are improvements that can and should be achieved using hook scripts. This is one.

**Comment by tbabej** [comment]
This feature was actually resurrected and implemented in 2.5.2 and 2.6.0.
